api/DTOs/UserDto.cs

```diff
@@ -2,11 +2,15 @@
 
 namespace YonderfulApi.DTOs
 {
-    public class EmployeeDto
+    public class UserDto
     {
         [Required]
         public string FirstName { get; set; }
         [Required]
         public string LastName { get; set; }
+        [Required]
+        public string Email { get; set; }
+        [Required]
+        public string Password { get; set; }
```

Comment:
For the password as well, make sure it has a min length. You can use the [StringLength] attribute here